Directory.Packages.props

```diff
@@ -113,9 +113,9 @@
     <PackageVersion Include="Yarp.ReverseProxy" Version="2.2.0" />
     <!-- Open Telemetry -->
     <PackageVersion Include="Npgsql.OpenTelemetry" Version="9.0.2" />
-    <PackageVersion Include="OpenTelemetry.Exporter.InMemory" Version="1.9.0" />
-    <PackageVersion Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.9.0" />
-    <PackageVersion Include="OpenTelemetry.Extensions.Hosting" Version="1.9.0" />
+    <PackageVersion Include="OpenTelemetry.Exporter.InMemory" Version="1.10.0" />
```

Comment:
Starting with 1.10 (there is a 1.11.1 btw) the net8.0 target depends on 

Microsoft.Extensions.Diagnostics.Abstractions (>= 9.0.0)
Microsoft.Extensions.Logging.Configuration (>= 9.0.0)
System.Diagnostics.DiagnosticSource (>= 9.0.0)

@eerhardt Do we need to split this for net8.0 and net9.0